lupa/_lupa.pyx

```diff
@@ -1868,9 +1868,9 @@ cdef int py_iter_next_with_gil(lua_State* L, py_object* py_iter) with gil:
 cdef int py_set_overflow_handler(lua_State* L) nogil:
     if not lua.lua_isnil(L, 1) and not lua.lua_isfunction(L, 1):
         return lua.luaL_argerror(L, 1, "expected nil or function")
-    lua.lua_pushlstring(L, LUPAOFH, LUPAOFH_LEN) # hdl ... name
-    lua.lua_pushvalue(L, 1)                      # hdl ... name hdl
-    lua.lua_rawset(L, lua.LUA_REGISTRYINDEX)     # hdl ...
+    lua.lua_pushlstring(L, LUPAOFH, len(LUPAOFH))  # hdl ... name
```

Comment:
This raises the following error:
`Calling gil-requiring function not allowed without gil`
I've opened an issue about the matter (https://github.com/cython/cython/issues/4093)
For the mean time, there is really no other way but to `DEF`-ine `len(LUPAOFH)` elsewhere.